src/ai/backend/manager/repositories/container_registry/admin_repository.py

```diff
@@ -28,47 +28,51 @@
 
 
 class AdminContainerRegistryRepository:
-    _repository: ContainerRegistryRepository
+    _db_source: ContainerRegistryDBSource
 
     def __init__(self, db: ExtendedAsyncSAEngine) -> None:
-        self._repository = ContainerRegistryRepository(db)
+        self._db_source = ContainerRegistryDBSource(db)
```

Comment:
All repositories that have the db source should be initialized with `ContainerRegistryDBSource` through the parameter rather than DB Engine object.